app/src/main/java/io/appium/uiautomator2/utils/ElementHelpers.java

```diff
@@ -84,8 +87,41 @@ private static AccessibilityNodeInfo elementToNode(Object element) {
      *
      * For example, appium returns elements like [{"ELEMENT":1}, {"ELEMENT":2}]
      */
-    public static JSONObject toJSON(AndroidElement el) throws JSONException {
-        return new JSONObject().put("ELEMENT", el.getId());
+    public static JSONObject toJSON(AndroidElement el) throws JSONException, UiObjectNotFoundException {
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put("ELEMENT", el.getId());
+        if (!Session.shouldUseCompactResponses()) {
+            for (String field : Session.getElementResponseFields()) {
+                try {
+                    if (field.equals("name")) {
+                        putNullable(jsonObject, field, el.getContentDesc());
+                    } else if (field.equals("text")) {
+                        putNullable(jsonObject, field, el.getText());
+                    } else if (field.equals("rect")) {
+                        putNullable(jsonObject, field, GetRect.getElementRectJSON(el));
+                    } else if (field.equals("enabled")) {
+                        putNullable(jsonObject, field, GetElementAttribute.getElementAttributeValue(el, field));
```

Comment:
we might use static import here to improve readability

Reply:
I'm not sure what you mean by this comment, can you elaborate?

Reply:
I mean `getElementAttributeValue` can be imported statically